libs/api/domains/fishing-license/src/graphql/fishing-license.resolver.ts

```diff
@@ -0,0 +1,41 @@
+import { UseGuards } from '@nestjs/common'
+import { Args, Query, Resolver } from '@nestjs/graphql'
+import { Audit } from '@island.is/nest/audit'
+import {
+  CurrentUser,
+  IdsAuthGuard,
+  IdsUserGuard,
+  ScopesGuard,
+} from '@island.is/auth-nest-tools'
+import type { User } from '@island.is/auth-nest-tools'
+import { Ship } from './models/ship.model'
+import { FishingLicense } from './models/fishing-license.model'
+import { FishingLicenseService } from '../lib/fishing-license.service'
+
+@UseGuards(IdsUserGuard, IdsAuthGuard, ScopesGuard)
+@Resolver()
+export class FishingLicenseResolver {
+  constructor(private fishingLicenseService: FishingLicenseService) {}
+
+  @Query(() => [Ship], {
+    name: 'ships',
```

Comment:
The name does not follow the graphql naming conventions for island.is.
This needs to be named `fishingLicenseShips`

ref: https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions

Reply:
Same goes for other query names and entities: like `Deprivation` should be called `FishingLicenseDeprivation`

Reply:
Will fix!